PcapDotNet/src/PcapDotNet.Packets/Dhcp/Options/DhcpBootfileNameOption.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PcapDotNet.Packets.Dhcp.Options
+{
+    /// <summary>
+    /// RFC 2132.
+    /// This option is used to identify a bootfile when the 'file' field in
+    /// the DHCP header has been used for DHCP options.
+    /// <pre>
+    ///  Code  Len    Bootfile name
+    /// +-----+-----+-----+-----+-----+---
+    /// | 67  |  n  |  c1 |  c2 |  c3 | ...
+    /// +-----+-----+-----+-----+-----+---
+    /// </pre>
+    /// </summary>
+    public class DhcpBootfileNameOption : DhcpStringOption
+    {
+        /// <summary>
+        /// create new DhcpBootfileNameOption
+        /// </summary>
+        /// <param name="bootfileName">Bootfilename</param>
+        public DhcpBootfileNameOption(string bootfileName) : base(bootfileName, DhcpOptionCode.BootfileName)
```

Comment:
I always tried to keep the names according the RFC specification. Only if it was not possible (spaces, conflict with existing names,...) I changed the name